### PR TITLE
[FLINK-26369][doc-zh] Translate the part zh-page mixed with not be translated.

### PR DESCRIPTION
**Pull request by GOODBOY008**
[FLINK-26369][doc-zh] Translate the part zh-page mixed with not be translated.

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@1996fanrui @Jiabao-Sun PTAL

**Comment by GOODBOY008** [comment]
@leonardBang PTAL , Thank you~

File: docs/content.zh/docs/deployment/ha/overview.md [hunk 1]
```diff
@@ -74,18 +74,9 @@ Flink 提供了两种高可用服务实现：
...
-Dirty entries are subject to cleanup, i.e. the corresponding job is either cleaned up by Flink at
-the moment or will be picked up for cleanup as part of a recovery. The entries will be deleted as
-soon as the cleanup succeeds. Check the JobResultStore configuration parameters under
-[HA configuration options]({{< ref "docs/deployment/config#high-availability" >}}) for further
-details on how to adapt the behavior.
+## 作业结果存储
 
+作业结果存储用于归档达到全局结束状态作业（即完成、取消或失败）的最终结果，其数据存储在文件系统上 (请参阅[job-result-store.storage-path]({{< ref "docs/deployment/config#job-result-store-storage-path" >}}))。
+只要没有正确清理相应的作业，此数据条目就是脏数据 (数据位于作业的子文件夹中 [high-availability.storageDir]({{< ref "docs/deployment/config#high-availability-storagedir" >}}))。
+脏数据是需要清理的，即通过相应的作业要么被 Flink 清理，要么将作为恢复的一部分被选取进行清理。清理成功后，数据条目将被删除。请参阅 [HA configuration options]({{< ref "docs/deployment/config#high-availability" >}}) 下作业结果存储的配置参数以获取有关如何调整行为的更多详细信息。
```
Comment (Jiabao-Sun) [inline docs/content.zh/docs/deployment/ha/overview.md:81]: 脏数据将被清理，即相应的作业要么在当前时刻被清理，要么在作业恢复过程中被清理。一旦清理成功，这些脏数据条目将被删除。

**Review by Jiabao-Sun — APPROVED** [review]

**Review by 1996fanrui — APPROVED** [review]
Thanks @GOODBOY008 for the translation and @Jiabao-Sun for the review!

LGTM assuming CI is green.

**Comment by 1996fanrui** [comment]
@flinkbot run azure

**Comment by GOODBOY008** [comment]
@flinkbot run azure

**Comment by 1996fanrui** [comment]
Hi @GOODBOY008 , would you mind rebasing the master branch? 

If I remember correctly, this ci isuue has been fixed in master branch.

**Comment by GOODBOY008** [comment]
@1996fanrui Branch has rebased, Thanks for your review.

**Review by 1996fanrui — APPROVED** [review]
CI is green, merging~

**Pull request merged**